tests/test_functional.py

```diff
@@ -424,13 +424,13 @@ def test_aggregations(new_resource, files):
     assert len(agg.files()) == file_count
     new_resource.aggregation_remove(agg)
     assert len(new_resource.aggregations()) == 0
-    assert len(new_resource.files()) == file_count
+    assert len(new_resource.files()) == file_count - 1  # exclude .vrt
```

Comment:
Refer changes I made to this test in my branch:
https://github.com/hydroshare/hsclient/blob/44-aggregation-data-processing-object/tests/test_functional.py#L397

Reply:
thanks updated with your changes